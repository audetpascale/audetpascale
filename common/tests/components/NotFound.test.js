/**
 * @jest-environment jsdom
 */
import React from "react";
import renderer from "react-test-renderer";
import { NotFound } from "../../index";

test("Generate NotFound", () => {
  const component = renderer.create(<NotFound>Facebook</NotFound>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
