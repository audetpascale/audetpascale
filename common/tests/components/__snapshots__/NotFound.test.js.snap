// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generate NotFound 1`] = `
<div
  className="css-19gncej-NotFound"
>
  <div
    className="css-1yrmf69"
  >
    <h1
      className="css-1wf7gym"
    >
      404 Fichier introuvable
    </h1>
    <span
      className="css-7axmvo-NotFound"
    >
      Je suis dÃ©solÃ©e,
       
      <span
        aria-label="sleeping emoji"
        role="img"
      >
        ðŸ˜”ðŸ’¤
      </span>
       
      je
       
      <big>
        <i>
          take a nap
        </i>
      </big>
      .
    </span>
    <span
      className="css-flryfv-NotFound"
    >
      <a
        aria-current="page"
        className="css-12xwyx6-NotFound"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        Accueil
      </a>
    </span>
  </div>
</div>
`;
